de.dlr.sc.virsat.model.edit/src/de/dlr/sc/virsat/model/concept/types/structural/IBeanStructuralElementInstance.java

```diff
@@ -237,4 +237,10 @@
 	 * @return boolean if there is a Category
 	 */
 	<BEAN_TYPE extends IBeanCategoryAssignment> boolean hasCategory(Class<BEAN_TYPE> catBeanClazz);
+	
+	/**
+	 * Checks if the current Bean SEI is a root element.

```

Comment:
But this is actually not what's being checked - you check if this SEI is of a SE that can be a root element. Let's say in CEF for a System under SystemOfSystems it will return true although it's not a root element